.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: BUG
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
```

Comment:
You can leave the description of what users should do in comments so that by default they're not rendered. You can see [here](https://raw.githubusercontent.com/scikit-learn/scikit-learn/master/PULL_REQUEST_TEMPLATE.md) an example, and the syntax looks like:

```
<!--
**Describe the bug**
A clear and concise description of what the bug is.
here comes the text which won't be rendered
-->
```

Reply:
I was wondering about that... would be annoying to have them displayed :-) Will address that...

Reply:
This is great!